templates/default/deploy-scripts/src/main.rs

```diff
@@ -0,0 +1,162 @@
+    let wasm_module: WasmModule = concordium_rust_sdk::common::from_bytes(&mut cursor)?;
+    Ok(wasm_module)
+}
+
+/// Command line flags.
+#[derive(clap::Parser, Debug)]
+#[clap(author, version, about)]
+struct App {
+    #[clap(
+        long = "node",
+        default_value = "http://node.testnet.concordium.com:20000",
+        help = "V2 API of the Concordium node."
+    )]
+    url:      v2::Endpoint,
+    #[clap(
+        long = "account",
+        help = "Location path and file name of the Concordium account key file (e.g. \
+                ./myPath/3PXwJYYPf6fyVb4GJquxSZU8puxrHfzc4XogdMVot8MUQK53tW.export)."
+    )]
+    key_file: PathBuf,
+    #[clap(
+        long = "modules",
+        help = "Location paths and names of Concordium smart contract modules. Use this flag \
+                several times if you have several smart contract modules to be deployed (e.g. \
+                --modules ./myPath/default.wasm.v1 --modules ./default2.wasm.v1)."
+    )]
+    modules:  Vec<PathBuf>,
+}
+
+/// Main function: It deploys to chain all wasm modules from the command line
+/// `--modules` flags. Write your own custom deployment/initialization script in
+/// this function. An deployment/initialization script example is given in this
+/// function for the `default` smart contract.
+#[tokio::main(flavor = "current_thread")]
+async fn main() -> Result<(), DeployError> {
+    let app: App = App::parse();
+
+    let concordium_client = v2::Client::new(app.url).await?;
+
+    let deployer = Deployer::new(concordium_client, &app.key_file)?;
+
+    let mut modules_deployed: Vec<ModuleDeployed> = Vec::new();
+
+    for contract in app.modules {
+        let wasm_module = get_wasm_module(contract.as_path())?;
+
+        let (_, module) = deployer.deploy_wasm_module(wasm_module, None).await?;
+
+        modules_deployed.push(module);
+    }
+
+    // Write your own deployment/initialization script below. An example is given
+    // here.
+
+    let param: OwnedParameter = OwnedParameter::default(); // Example
+
+    let init_method_name: &str = "init_default"; // Example
+
+    let payload = InitContractPayload {
+        init_name: OwnedContractName::new(init_method_name.into())?,
+        amount: Amount::from_micro_ccd(0),
+        mod_ref: modules_deployed[0].module_ref,
+        param,
+    }; // Example
+
+    let (_, contract) = deployer.init_contract(payload, None, None).await?; // Example
+
+    let bytes = contracts_common::to_bytes(&false); // Example
+
+    let update_payload = transactions::UpdateContractPayload {
+        amount:       Amount::from_ccd(0),
+        address:      contract,
+        receive_name: OwnedReceiveName::new_unchecked("default.receive".to_string()),
+        message:      bytes.try_into()?,
+    }; // Example
+
+    let mut energy = deployer.estimate_energy(update_payload.clone(), None).await?; // Example
+
+    // We add 100 energy to be save.
+    energy.energy += 100; // Example
```

Comment:
Adding some energy seems reasonable, so I think we should recommend it in the docs of `estimate_energy` 😊